README.md

```diff
@@ -46,6 +46,10 @@ If you use Mouse1, Rat1, Zebrafish1, Fruitfly1 or Worm1 in your research, please
 
 This repository is administered by Jonathan Robinson ([@JonathanRob](https://github.com/jonathanrob)) and Hao Wang ([@Hao-Chalmers](https://github.com/hao-chalmers)), Division of Systems and Synthetic Biology, Department of Biology and Biological Engineering, Chalmers University of Technology.
 
+## Contributing
+
+Contributions are always welcome! Please read the [Contributing guidelines](https://github.com/SysBioChalmers/Human-GEM/blob/main/.github/CONTRIBUTING.md) before start.
+
```

Comment:
This move was due to following intentions:
- Human-GEM is driven by contributors who usually expect to see this guideline in a more visible place (on top) in README.
- Duplication is useful sometimes, to highlight the importance of contributors

Reply:
In that case, I would be tempted to still merge the sections so that information is centralized, but add a pointer to this in the top-most part of the Readme, eg

> This repository contains the latest version of Human-GEM, a human genome-scale metabolic model.
> We encourage [contributions]().

Reply:
pointer is added